migration/sql-files/005-add-search-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS fields_title ON work_items USING GIN (to_tsvector('english', fields->>'system.title'));
```

Comment:
None of these indexes are hit by the query that is generated. 

```sql
EXPLAIN ANALYZE SELECT count(*) over () as cnt2 , * FROM "work_items"
WHERE "work_items".deleted_at IS NULL
AND ((
	setweight(to_tsvector('english',coalesce(fields->>'system.title','')),'B')||
	setweight(to_tsvector('english',coalesce(fields->>'system.description','')),'C')|| 
	setweight(to_tsvector('english', id::text),'A')
	@@ to_tsquery('english','"aslak"')
)) LIMIT 100

#	QUERY PLAN
1	Limit  (cost=0.00..17565.50 rows=100 width=626) (actual time=12492.779..12492.789 rows=20 loops=1)
2	  ->  WindowAgg  (cost=0.00..115932.30 rows=660 width=626) (actual time=12492.778..12492.784 rows=20 loops=1)
3	        ->  Seq Scan on work_items  (cost=0.00..115924.05 rows=660 width=626) (actual time=2024.570..12492.751 rows=20 loops=1)
4	              Filter: ((deleted_at IS NULL) AND (((setweight(to_tsvector('english'::regconfig, COALESCE((fields ->> 'system.title'::text), ''::text)), 'B'::"char") || setweight(to_tsvector('english'::regconfig, COALESCE((fields ->> 'system.description'::text), ''::text)), 'C'::"char")) || setweight(to_tsvector('english'::regconfig, (id)::text), 'A'::"char")) @@ '''aslak'''::tsquery))
5	              Rows Removed by Filter: 132110
6	Planning time: 0.136 ms
7	Execution time: 12492.825 ms
``` 

That is 12s execution time on my dataset, vs 5ms if it hit the index :)